docs/design/autoware-interfaces/ad-api/types/autoware_adapi_v1_msgs/msg/DoorLayout.md

```diff
@@ -0,0 +1,19 @@
+---
+# This file is generated by tools/autoware-interfaces/generate.py
+title: autoware_adapi_v1_msgs/msg/DoorLayout
+used:
+  - autoware_adapi_v1_msgs/srv/GetDoorLayout
+---
+
+{% extends 'design/autoware-interfaces/templates/autoware-data-type.jinja2' %}
+{% block definition %}
+
+```txt
+uint8 GET_ON = 1
+uint8 GET_OFF = 2
+
+uint8[] roles
+geometry_msgs/Pose pose
```

Comment:
> Is the position really needed? Maybe door id (or name) might be better if you only want to specify which door to open.

As you say, the name is enough, so I will change it.

Reply:
I talked with @mitsudome-r -san. The name "name" gives the impression that the system can be used for control. So rename it to "description" to avoid that.